hpcgap/lib/init.g

```diff
@@ -1052,15 +1093,8 @@ end);
 HELP_ADD_BOOK("Tutorial", "GAP 4 Tutorial", "doc/tut");
 HELP_ADD_BOOK("Reference", "GAP 4 Reference Manual", "doc/ref");
 HELP_ADD_BOOK("Changes", "Changes from Earlier Versions", "doc/changes");
-HELP_ADD_BOOK("HPC-GAP", "HPC-GAP Shared Memory Extensions and MPI", "doc/hpc");
-
-
-#############################################################################
-##
-##  MPIGAP loading 
-##
-if IsBoundGlobal("MPI_Initialized") then
-  ReadLib("distributed/distgap.g");
+if IsBound(HPCGAP) then
+  HELP_ADD_BOOK("HPC-GAP", "HPC-GAP Shared Memory Extensions and MPI", "doc/hpc");
```

Comment:
Please fix the 2nd argument - there is no MPI there any more.

Reply:
Good catch, fixed